### PR TITLE
cli: added experimental create-github-app command

### PR DESCRIPTION
**Pull request by jhaals**
## Hey, I just made a Pull Request!
The new command automates the creation of GitHub Apps.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

**Review by benjdlambert — APPROVED** [review]

File: packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+const MANIFEST_DATA = {
+  default_events: ['create', 'delete', 'push', 'repository'],
+  default_permissions: {
+    contents: 'read',
+    metadata: 'read',
+  },
+  name: 'Backstage-<changeme>',
+  url: 'https://backstage.io',
+  description: 'Github App for Backstage',
```
Comment (freben) [inline packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts:30]:
```suggestion
  description: 'GitHub App for Backstage',
```

:)

File: packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  privateKey: string;
+};
+
+export class GithubCreateAppServer {
+  private baseUrl?: string;
+  private webhookUrl?: string;
+
+  static async run({ org }: { org: string }): Promise<GithubAppConfig> {
+    const encodedOrg = encodeURIComponent(org);
+    const actionUrl = `https://github.com/organizations/${encodedOrg}/settings/apps/new`;
```
Comment (adamdmharvey) [inline packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts:66]: Maybe valuable to throw in a `// TODO` to note that this experimental command at this point does not support GitHub Enterprise? (not suggesting it should yet, but as it iterates, future iterations could either prompt for a GHE host or potentially call out to look for an app-config.yaml)

Comment (jhaals) [reply packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts:66]: GHE is definitely something we want to support in the CLI but it wasn't possible to post the manifest file unfortunately.

Comment (jhaals) [reply packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts:66]: @adamdmharvey I've added a comment by the command entrypoint. Please take a look :)

Comment (adamdmharvey) [reply packages/cli/src/commands/create-github-app/GithubCreateAppServer.ts:66]: merci @jhaals ! 😊

**Review by Rugvip — APPROVED** [review]

**Pull request merged**